parsons/action_network/action_network.py

```diff
@@ -51,7 +51,7 @@ def _get_entry_list(self, object_name, limit=None, per_page=25, filter=None):
         while True:
             response = self._get_page(object_name, page, per_page, filter=filter)
             page = page + 1
-            response_list = response["_embedded"][f"osdi:{object_name}"]
+            response_list = response["_embedded"][list(response["_embedded"])[0]]
```

Comment:
Can you explain what's going on here? I want to make sure this isn't a breaking change.

Reply:
Thank you for the explanation! Super helpful. So it sounds like this will not break any existing code, because only `object_name`s like "donations" was working anyway, and your approach still works for them. This just expands the valid object_names. And I didn't realize this before, but `object_name` really only seems to be used internally, and is not exposed to the end-user, so that makes worrying about breaking changes way less important anyway. :)